internal/spdxlicense/license_list.go

```diff
@@ -472,49 +472,49 @@ var licenseIDs = map[string]string{
 	"fsfullr":                                  "FSFULLR",
 	"ftl":                                      "FTL",
 	"gd":                                       "GD",
-	"gfdl-1":                                   "GFDL-1.1",
+	"gfdl-1":                                   "GFDL-1.1-only",
```

Comment:
Does adding `-only` here change the semantics? This seems like a reasonable thing to assume, but it would be good to be sure. Same question for all the other `-only` additions above and below.